panic: runtime error: index out of range

Sorry for the redacted stack trace. 

I got a panic in what is essentially this line: https://github.com/flynn/go-tuf/blob/890a6cb82044de20e094222d137721d287f46b71/local_store.go#L265
I get that this is probably a case of some wrong input of sorts (e.g. name being empty or `nil`), but the program should still not crash here.

panic: runtime error: index out of range

```
goroutine 1 [running]:
[github.com/flynn/go-tuf.(*fileSystemStore).Commit.func3(0xc0003763e3](https://github.com/flynn/go-tuf.(*fileSystemStore).Commit.func3(0xc0003763e3), 0xc, 0xc000376301)
        /<readacted>/src/github.com/flynn/go-tuf/local_store.go:274 +0x1a2
[github.com/flynn/go-tuf.(*fileSystemStore).Commit.func4(0xc000376380](https://github.com/flynn/go-tuf.(*fileSystemStore).Commit.func4(0xc000376380), 0x6f, 0x86d0e0, 0xc00037fd40, 0x0, 0x0, 0x4b732a, 0xc00037fd40)
```

--- a/local_store.go
+++ b/local_store.go
@@ -262,7 +262,7 @@ func (f *fileSystemStore) Commit(meta map[string]json.RawMessage, consistentSnap
 		if consistentSnapshot {
 			// strip out the hash
 			name := strings.SplitN(filepath.Base(path), ".", 2)
-			if name[1] == "" {
+			if len(name) != 2 || name[1] == "" {
 				return false
 			}
 			path = filepath.Join(filepath.Dir(path), name[1])

